### PR TITLE
show profile indicator description on mapchip

### PR DESCRIPTION
**Pull request by waracci**
## Description

The description on map chip on data mapper should come from profile indicator as with rich data view, not from the dataset. 

## How to test it locally

I located a profile indicator with no description and the map chip does not show any description.

Screenshot from 2021-06-25 10-30-41

I went to the staging admin panel and added a test description on the profile indicator.
Example: https://staging.wazimap-ng.openup.org.za/admin/profile/profileindicator/?q=Population+By+Gender&profile__name=Youth+Explorer

The description shows up on the map chip as expected.

Screenshot from 2021-06-25 10-29-58

## Checklist

- [ ]  🚀 is the code ready to be merged and go live?
- [ ]  🛠 does it work (build) locally
- [ ] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [ ]  📰 good title
- [ ]  📝good description
- [ ]  🔖 issue linked
- [ ]  📖 changelog filled out
- [ ] commit messages are meaningful

### Code Quality

- [ ]  🚧 no commented out code
- [ ]  🖨 no unnecessary logging
- [ ]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [ ]  ✅ ran tests locally & are passing

### CODE REVIEW

**Comment by waracci** [comment]
[description on map chip on data mapper should come from profile indicator as with rich data view, not from dataset. [Timebox 0.5 days]](https://trello.com/c/XY5PlvLJ/806-description-on-map-chip-on-data-mapper-should-come-from-profile-indicator-as-with-rich-data-view-not-from-dataset-timebox-05-day)

**Comment by michaelglenister** [comment]
So args.description is a description of the dataset, while args.data.description is of the indicator. Changes look good, thanks

File: src/js/elements/mapchip.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class MapChip extends Component {
         $(mapOptionsClass).show();  //webflow.js adds display:none when clicked on x
         $(mapOptionsClass).find('.filters__header_close').on('click', () => this.removeMapChip());
 
-        $('.map-options__context .map-option__context_text div').text(args.description);
+        $('.map-options__context .map-option__context_text div').text(args.data.description);
```
Comment (michaelglenister) [inline src/js/elements/mapchip.js:43]: Good

File: src/js/profile/chart.js [hunk 2]
```diff
@@ -271,15 +271,15 @@ export class Chart extends Component {
...
+                    'csv': self.exportAsCsv,
+                    'excel': self.exportAsExcel,
+                    'json': self.exportAsJson,
+                    'kml': self.exportAsKml,
+                };
+                self.triggerEvent(`profile.chart.download_${exportType}`, self);
 
                 let fileName = self.getChartTitle('-');
-                downloadFn.fn(fileName);
+                downloadFn[exportType](fileName);
```
Comment (michaelglenister) [inline src/js/profile/chart.js:282]: This is the fix included from https://github.com/OpenUpSA/wazimap-ng-ui/pull/416
Reminder to close that PR when this is merged

Comment (waracci) [reply src/js/profile/chart.js:282]: This comment is meant for @adieyal

Comment (jbothma) [reply src/js/profile/chart.js]: github will actually do that automatically

**Pull request merged**